include/seqan3/alphabet/structure/wuss.hpp

```diff
@@ -0,0 +1,443 @@
+    };
+
+    //!\brief Value-to-char conversion table.
+    static constexpr std::array<char_type, value_size> value_to_char
+    {
+        [] () constexpr
+        {
+            std::array<char_type , value_size> chars
+            {
+                '.', '<', '>', ':', ',', '-', '_', '~', ';', '(', ')', '[', ']', '{', '}'
+            };
+
+            // pseudoknot letters
+            for (rank_type rnk = 15u; rnk + 1u < value_size; rnk += 2u)
+            {
+                char_type const off = static_cast<char_type>((rnk - 15u) / 2u);
+                chars[rnk] = 'A' + off;
+                chars[rnk + 1u] = 'a' + off;
+            }
+
+            return chars;
+        } ()
+    };
+
+    //!\brief Char-to-value conversion table.
+    static constexpr std::array<internal_type, 256> char_to_value
+    {
+        [] () constexpr
+        {
+            std::array<internal_type, 256> rank_table{};
+
+            // initialize with unpaired (std::array::fill unfortunately not constexpr)
+            for (internal_type & rnk : rank_table)
+                rnk = internal_type::UNPAIRED6;
+
+            // set alphabet values
+            for (rank_type rnk = 0u; rnk < value_size; ++rnk)
+                rank_table[value_to_char[rnk]] = static_cast<internal_type>(rnk);
+            return rank_table;
+        } ()
+    };
+
+public:
+    //!\privatesection
+    //!\brief The data member.
+    internal_type _value;
+    //!\publicsection
+};
+
+template <uint8_t SIZE>
+constexpr wuss<SIZE> wuss<SIZE>::UNPAIRED{internal_type::UNPAIRED};
+
+template <uint8_t SIZE>
+constexpr wuss<SIZE> wuss<SIZE>::PAIR_OPEN{internal_type::PAIR_OPEN};
+
+template <uint8_t SIZE>
+constexpr wuss<SIZE> wuss<SIZE>::PAIR_CLOSE{internal_type::PAIR_CLOSE};
+
+template <uint8_t SIZE>
+constexpr wuss<SIZE> wuss<SIZE>::UNPAIRED1{internal_type::UNPAIRED1};
+
+template <uint8_t SIZE>
+constexpr wuss<SIZE> wuss<SIZE>::UNPAIRED2{internal_type::UNPAIRED2};
+
+template <uint8_t SIZE>
+constexpr wuss<SIZE> wuss<SIZE>::UNPAIRED3{internal_type::UNPAIRED3};
+
+template <uint8_t SIZE>
+constexpr wuss<SIZE> wuss<SIZE>::UNPAIRED4{internal_type::UNPAIRED4};
+
+template <uint8_t SIZE>
+constexpr wuss<SIZE> wuss<SIZE>::UNPAIRED5{internal_type::UNPAIRED5};
+
+template <uint8_t SIZE>
+constexpr wuss<SIZE> wuss<SIZE>::UNPAIRED6{internal_type::UNPAIRED6};
+
+template <uint8_t SIZE>
+constexpr wuss<SIZE> wuss<SIZE>::PAIR_OPEN1{internal_type::PAIR_OPEN1};
+
+template <uint8_t SIZE>
+constexpr wuss<SIZE> wuss<SIZE>::PAIR_CLOSE1{internal_type::PAIR_CLOSE1};
+
+template <uint8_t SIZE>
+constexpr wuss<SIZE> wuss<SIZE>::PAIR_OPEN2{internal_type::PAIR_OPEN2};
+
+template <uint8_t SIZE>
+constexpr wuss<SIZE> wuss<SIZE>::PAIR_CLOSE2{internal_type::PAIR_CLOSE2};
+
+template <uint8_t SIZE>
+constexpr wuss<SIZE> wuss<SIZE>::PAIR_OPEN3{internal_type::PAIR_OPEN3};
+
+template <uint8_t SIZE>
+constexpr wuss<SIZE> wuss<SIZE>::PAIR_CLOSE3{internal_type::PAIR_CLOSE3};
+
+//!Alias for the default type wuss51.
```

Comment:
I think you forgot a `\brief`!
Please double-check that your files produce no warnings in doxygen!